packages/flutter/test/painting/gradient_test.dart

```diff
@@ -191,6 +191,32 @@ void main() {
     ));
   });
 
+  test('LinearGradient lerp test with transforms', () {
+    const LinearGradient testGradient1 = LinearGradient(
+      transform: GradientRotation(math.pi/4),
+      colors: <Color>[
+        Color(0x33333333),
+        Color(0x66666666),
+      ],
+      stops: <double>[0, 1],
+    );
+    const LinearGradient testGradient2 = LinearGradient(
+      transform: GradientRotation(math.pi/2),
+      colors: <Color>[
+        Color(0x33333333),
+        Color(0x66666666),
+      ],
+      stops: <double>[0, 1],
+    );
+
+    final LinearGradient? actual0 = LinearGradient.lerp(testGradient1, testGradient2, 0.0);
+    final LinearGradient? actual1 = LinearGradient.lerp(testGradient1, testGradient2, 1.0);
+    final LinearGradient? actual2 = LinearGradient.lerp(testGradient1, testGradient2, 0.5);
+    expect(testGradient1, equals(actual0));
+    expect(testGradient2, equals(actual1));
+    expect(testGradient2, equals(actual2));
```

Comment:
```suggestion
    expect(actual0, equals(testGradient1));
    expect(actual1, equals(testGradient2));
    expect(actual2, equals(testGradient2));
```

Tiny nit: "actual" should be passed as the first argument for [`expect()`](https://github.com/flutter/flutter/blob/7b6d667701e38c1d9d700e2bcc29bc7a06598c59/packages/flutter_test/lib/src/widget_tester.dart#L458-L475) (here and below). Not a big deal though, since in this case the test works fine either way.